### PR TITLE
New repository providing an example implementation of plugin

### PR DESCRIPTION
**Pull request by laurentganne**
New repository providing an example implementation of plugin as described in Yorc documentation at https://yorc.readthedocs.io/en/latest/plugins.html

### CODE REVIEW

File: src/go.mod [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module github.com/src
+
+require (
+	github.com/ystia/yorc v3.1.2+incompatible
```
Comment (loicalbertin) [inline src/go.mod]: Why do we have this ?

File: src/go.mod [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	github.com/ystia/yorc/v3 v3.2.0-M2
```
Comment (loicalbertin) [inline src/go.mod]:
```suggestion
	github.com/ystia/yorc/v3 v3.2.0-M4
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+This repository provides an example of plugin extending the Ystia orchestrator, as described in [Yorc documentation on plugins](https://yorc.readthedocs.io/en/latest/plugins.html).
```
Comment (loicalbertin) [inline README.md:5]:
```suggestion
# Yorc Plugin example

This repository provides an example of plugin extending the Ystia orchestrator, as described in [Yorc documentation on plugins](https://yorc.readthedocs.io/en/latest/plugins.html).
```

File: src/go.mod (hunk 1 above)
Comment (laurentganne) [reply src/go.mod]: generated automatically, don't know why, fixed as described below in next comment

File: src/go.mod (hunk 2 above)
Comment (laurentganne) [reply src/go.mod]: done

File: README.md (hunk 3 above)
Comment (laurentganne) [reply README.md:5]: ok, added a title

**Review by loicalbertin — APPROVED** [review]
LGTM :+1: :clap:

**Pull request merged**